src/vm.rs

```diff
@@ -15,6 +15,7 @@ use crate::codegen::CompiledFun;
 use crate::deepcopy::{deepcopy, remap};
 use crate::host::*;
 use crate::str::Str;
+use rustc_hash::FxHashMap as HashMapDefault;
```

Comment:
Only thing I would say is: can we just do
`use rustc_hash::FxHashMap as HashMap` ?

That would minimize the diff, and honestly this codebase should probably be using this hash map in other places too.